source/runbooks/oracle-licence-discovery.html.md.erb

```diff
@@ -0,0 +1,85 @@
+   - The automation documents are stored in the "documents" section under the self owned tab.
+
+   ![automation documents location ](images/documents_section.png)
+
+   - To view the automation, click on the automation section on the left hand side and click on the execution id next to the runbook name of "OracleDbLTS-Orchestrate" from here you will be able to see the successful and failed executions and dig down further.
+
+   ![automation location ](images/automation_section.png)
+
+   - To find the state associated with automation, go to the "State Manager" section under Systems Manager this will show you the last run state of the automation and when the frequency of the automation, you can also apply the association from here to force a run of the automation.
+   
+   ![state manager location ](images/state_manager_section.png)
+
+ <%= warning_text('The current run of the automation is set to run once every week, this can be changed by creating a PR in the [AWS root account repo](https://github.com/ministryofjustice/aws-root-account/blob/main/organisation-security/terraform/license-manager.tf). This will need to be reviewed by the #ask-root-account team.
+Currently the automation is restricted to three organization accounts, this can be changed by creating a PR in the [AWS root account repo](https://github.com/ministryofjustice/aws-root-account/blob/main/organisation-security/terraform/locals.tf). Amending the locals file to include the organization OU that you want to include in the automation.
+') %>
+### Athena
+
+Athena is an interactive query service that makes it easy to analyse data in Amazon S3 using standard SQL. Athena is serverless, so there is no infrastructure to manage, and you pay only for the queries that you run.
+   - since the is data is synchronised from the discovery process to an s3 bucket we can use Athena to query the data using sql.
+   - Use the following SQL script to bring back the data into a table that can be exported:
+   
+   ```
+
+   SELECT * FROM "ssm_resource_sync"."custom_oraclefeatureusagedetails"
+   LEFT JOIN "ssm_resource_sync"."aws_instancedetailedinformation"
+   ON "ssm_resource_sync"."custom_oraclefeatureusagedetails"."resourceid" = "ssm_resource_sync"."aws_instancedetailedinformation"."resourceid"
+   LEFT JOIN "ssm_resource_sync"."custom_oracledatabaseproduct"
+   ON "ssm_resource_sync"."custom_oraclefeatureusagedetails"."resourceid" = "ssm_resource_sync"."custom_oracledatabaseproduct"."resourceid"
+   LEFT JOIN "ssm_resource_sync"."aws_tag"
+   ON "ssm_resource_sync"."custom_oraclefeatureusagedetails"."resourceid" = "ssm_resource_sync"."aws_tag"."resourceid"
+   WHERE "ssm_resource_sync"."aws_tag"."key" LIKE 'business-unit'
+
+   ```
+The results will look like the following:
+![Athena results](images/athena_results.png)
+
+## Benefits
+   Compliance: Ensure adherence to Oracle licensing terms.
```

Comment:
licensing but you use licence  everywhere else

Reply:
changes all the others to license or licensing